flask_api/app/routes.py

```diff
@@ -21,11 +21,13 @@
 
 
 @api.route("/healthchecker", methods=["GET"])
+@cross_origin()
```

Comment:
`@cross_origin` is everywhere right now. Did you try the following?
```python
api = Blueprint("api", __name__)
CORS(api)

@api.route("/healthchecker", methods=["GET"])
def healthchecker():
    ...
```
See [here](https://flask-cors.readthedocs.io/en/latest/api.html#using-cors-with-blueprints)